third_party/js.bzl

```diff
@@ -525,25 +525,39 @@ def tensorboard_js_workspace():
+                "https://raw.githubusercontent.com/google/material-design-icons/b3f05bfbf4329a5b63f50a720f867c2bac163f98/src/content/push_pin/materialiconsoutlined/24px.svg",
+            ],
         },
         rename = {
+            "ic_arrow_downward_24px.svg": "arrow_downward_24px.svg",
+            "ic_arrow_upward_24px.svg": "arrow_upward_24px.svg",
+            "ic_cancel_24px.svg": "cancel_24px.svg",
+            "ic_chevron_left_24px.svg": "chevron_left_24px.svg",
+            "ic_chevron_right_24px.svg": "chevron_right_24px.svg",
+            "ic_clear_24px.svg": "clear_24px.svg",
             "ic_content_copy_24px.svg": "content_copy_24px.svg",
+            "ic_error_24px.svg": "error_24px.svg",
+            "ic_expand_less_24px.svg": "expand_less_24px.svg",
+            "ic_expand_more_24px.svg": "expand_more_24px.svg",
+            "ic_flag_24px.svg": "flag_24px.svg",
+            "ic_get_app_24px.svg": "get_app_24px.svg",
             "ic_help_outline_24px.svg": "help_outline_24px.svg",
             "ic_info_outline_24px.svg": "info_outline_24px.svg",
+            "ic_push_pin_24px.svg": "push_pin_24px.svg",
+            "ic_push_pin_outline_24px.svg": "push_pin_outline_24px.svg",
             "ic_refresh_24px.svg": "refresh_24px.svg",
             "ic_settings_24px.svg": "settings_24px.svg",
             "ic_search_24px.svg": "search_24px.svg",
-            "ic_error_24px.svg": "error_24px.svg",
-            "ic_chevron_left_24px.svg": "chevron_left_24px.svg",
-            "ic_chevron_right_24px.svg": "chevron_right_24px.svg",
             "ic_visibility_off_24px.svg": "visibility_off_24px.svg",
-            "ic_flag_24px.svg": "flag_24px.svg",
-            "ic_clear_24px.svg": "clear_24px.svg",
-            "ic_expand_more_24px.svg": "expand_more_24px.svg",
-            "ic_expand_less_24px.svg": "expand_less_24px.svg",
-            "ic_cancel_24px.svg": "cancel_24px.svg",
-            "ic_arrow_downward_24px.svg": "arrow_downward_24px.svg",
-            "ic_arrow_upward_24px.svg": "arrow_upward_24px.svg",
-            "ic_get_app_24px.svg": "get_app_24px.svg",
+            "http://mirror.tensorflow.org/raw.githubusercontent.com/google/material-design-icons/b3f05bfbf4329a5b63f50a720f867c2bac163f98/src/content/push_pin/materialicons/24px.svg": "keep_24px.svg",
+            "https://raw.githubusercontent.com/google/material-design-icons/b3f05bfbf4329a5b63f50a720f867c2bac163f98/src/content/push_pin/materialicons/24px.svg": "keep_24px.svg",
+            "http://mirror.tensorflow.org/raw.githubusercontent.com/google/material-design-icons/b3f05bfbf4329a5b63f50a720f867c2bac163f98/src/content/push_pin/materialiconsoutlined/24px.svg": "keep_outline_24px.svg",
+            "https://raw.githubusercontent.com/google/material-design-icons/b3f05bfbf4329a5b63f50a720f867c2bac163f98/src/content/push_pin/materialiconsoutlined/24px.svg": "keep_outline_24px.svg",
```

Comment:
The rename attribute map's keys must either be the "after the last slash piece", or the full URL.

https://github.com/bazelbuild/rules_closure/blob/master/closure/filegroup_external.bzl#L167-L170